src/components/LandingPage.tsx

```diff
@@ -4,53 +4,122 @@ import { Image } from 'semantic-ui-react';
 import timage from '../images/landing-background-top.png';
 import bimage from '../images/landing-background-bot.png';
 
-const LandingPage = ({ user }: { user: string }): any => {
+const LandingPage = ({
+  user,
+  groups,
+  role,
+  rolename,
+}: {
+  user: string;
+  groups: any;
+  role: string;
+  rolename: string;
+}): any => {
+  const isVolunteer = groups && groups.includes('Volunteer');
```

Comment:
It will be most convenient to use the `hasPermission` utility to check for volunteers. See here for an example:

https://github.com/cusac/mbt.guide/blob/e81cef8e003e31e0b259652da9ca12457e0213d4/src/components/SegmentViewer.tsx#L24

By default we are not including the user.groups object since it does not contain information that we need for now. The `Volunteer` string will be included in the user's scope as long as they are a part of that group.

You can see these docs for more insight into user scopes if you like:

https://github.com/JKHeadley/appy-backend/wiki/Authorization

Note that the permission system we are using is based off of that repository.

Reply:
It is done. Thanks a lot. Its simpler. We have separate pages now.